terraform-infrastructure-pipelines-workshop/verify-deployments.md

```diff
@@ -85,15 +88,18 @@ Katacoda environment, allowing you to access it from within Katacoda.
 
 `kubectl --kubeconfig=kubeconfig -n hashicorp-learn port-forward --address 0.0.0.0 consul-server-0 8500:8500`{{execute T1}} 
 
-Click on the "Consul UI" tab in your terminal pane or visit `[[HOST_SUBDOMAIN]]-8500-[[KATACODA_HOST]].environments.katacoda.com` in your browser to access the Consul UI.
+Click on the "Consul UI" tab in your terminal pane or visit
+`https://[[HOST_SUBDOMAIN]]-8500-[[KATACODA_HOST]].environments.katacoda.com` in
+your browser to access the Consul UI.
 
 > **Note:** The Consul UI does not show Vault in the list of services because
   its service_registration stanza in the Helm chart defaults to Kubernetes.
   However, Vault is still configured to use Consul as a backend.
 
 ## Access Vault UI
 
-Open a new tab in the Katacoda terminal pane using the "+" symbol.
+Open a new tab in the Katacoda terminal pane by using the "+" symbol and
```

Comment:
You don't have to do this because clicking the command will automatically open it up in a new tab